src/lib/chat/matrix-client.ts

```diff
@@ -1800,47 +1810,6 @@ export class MatrixClient implements IChatClient {
     return this.matrix.getDeviceId();
   }
 
-  async shareHistoryKeys(roomId: string, userIds: string[]) {
-    // This resolves some instances where the other device is missing an old key from the room
-    this.debug('sending shared history keys', roomId, userIds);
-    await this.matrix.sendSharedHistoryKeys(roomId, userIds);
-    this.debug('done sending shared history keys');
-  }
-
-  async cancelAndResendKeyRequests() {
-    // It seems like this may already be happening automatically when we have
-    // problems decrypting messages.
-    this.debug('cancelling and resending key requests');
-    await this.matrix.crypto?.cancelAndResendAllOutgoingKeyRequests();
-    this.debug('done cancelling and resending key requests');
-  }
-
-  async discardOlmSession(roomId: string) {
-    // Throw away the olm session for the room...does this automatically
-    // regenerate or do we need the resetOlmSession call below?
-    this.debug('discarding session', roomId);
-    await this.matrix.getCrypto().forceDiscardSession(roomId);
-    this.debug('done discarding session', roomId);
-  }
-
-  async resetOlmSession(roomId: string) {
-    // RESET THE OLM SESSION
-    // Unsure which errors this might resolve. It seems like once you've missed
-    // something related to olm that you can't recover from it and this may only
-    // help with future messages.
-    this.debug('resetting the olm session', roomId);
-    this.matrix.getCrypto().forceDiscardSession(roomId);
-    const room = this.matrix.getRoom(roomId);
-    const members = (await room?.getEncryptionTargetMembers()) || [];
-    if (members.length > 0) {
-      await this.matrix.crypto?.ensureOlmSessionsForUsers(
-        members.map((m) => m.userId),
-        true
-      );
-    }
-    this.debug('done resetting the olm session', roomId);
-  }
-
```

Comment:
These were all using removed methods, so you'll see them pulled out throughout the codebase.
They were just used for a dev panel, so shouldn't affect any users.

Reply:
yeah correct this should be fine